### DISCUSSION TITLE
Does MAUI support App Shortcuts and App Intents

### DISCUSSION THREAD

**Discussion by MortenGuld** [dotnet/maui#19130] [General]
I would like to use Siri with my app.

Any information is welcome, as I have not been able to find anything regarding this topic. 

Thanks, 
Morten

**Comment by asllcpra** [comment]
We are also moving an app to net MAUI IOS from Xamarin.IOS and would like to support App Intents. Per https://developer.apple.com/documentation/appintents/appintentspackage you can package them in a swift framework, but we have been unable to find a way to configure the NET MAUI IOS project to link the intentions. 

`struct MyAppPackage: AppIntentsPackage {
   static var includedPackages: [any AppIntentsPackage.Type] {
       [MyFrameworkPackage.self]
   }
}` need something similar in C# to expose Intents in the app bundle. Any Ideas ?

**Comment by Caslle43** [comment]
Are there any news on this topic ? I'm trying to use AppIntents as well as SiriShortcuts in my Maui-App. Thanks in advance.

**Comment by Ievhenii365** [comment]
Hi, guys, does maui really support using app intents as siri shortcuts with voice control?
